scripts/spack/configs/linux_ubuntu_24/spack.yaml

```diff
@@ -0,0 +1,232 @@
+spack:
```

Comment:
this reminds me, will docker x64_86 images be maintainable, now that we're both on apple-silicon macs?